index.template.html

```diff
@@ -3916,6 +3925,191 @@ <h5>Inferencing of Personally Identifiable Information Privacy Risk</h5>
 
   </section>
 
+  <section id="thing-templates" class="normative">
+  <h1>Thing Model</h1>
+      
+  <p class="note">
+    The following section has its origin in [[wot-thing-description]], Annex C. Here Thing Description 
+    Template is renamed to <a>Thing Model</a>, but keeps the same intention.
+    For this version of the specification, <a>Thing Model</a> and its model features (e.g., extendions, obligations, placeholder) are formal 
+    introduced.
+  </p>
+
+    <p>
+    A <em>Thing Model</em> is a special kind of a <a>Thing Description</a> with less restriction as 
+    driven in <a href="#sec-vocabulary-definition"></a> and <a href="#sec-td-serialization"></a>. Thing Model samples can 
+    also be seen as a subset of the <a>Thing Descriptions</a> information model, which may not contain, 
+    for example, concrete protocol and security information.
+    </p>
+    <p>
+    The Thing Model enables:
+    <ul>
+    <li> onboard and manage of multiple <a>Thing</a> models, e.g., by a cloud service. </li>
+    <li> simulate of devices/<a>Things</a> that have not yet been developed.</li>
+    <li> develop common applications across devices from different manufacturers that share a common <a>Thing</a> model.</li>
+    <li> combining multiple models into a <a>Thing</a>.
+    <li> implementation support of a concrete <a>Thing</a>.
+    </ul>
+    </p>
+    <p>
+    The Thing Model is a logical description of the interface and possible interaction with <a>Thing's</a>
+    <a>Properties</a>, <a>Actions</a>, and <a>Events</a>, however it does not contain <a>Thing</a> instance-specific
+    information, such as security information and concrete protocol usage, or even a serial number and GPS location.
+    </p>
+    
+    <p>
+    A <a>Thing Model</a> is recognized by the top level <em>@type</em>.
+    <span class="rfc2119-assertion" id="td-model-identification">
+        <a>Thing Model</a> definitions MUST use the keyword <em>@type</em> at top level 
+        and a value of type string or array that equals or respectively contains <code>ThingModel</code>.
+    </span>
+    </p>
+
+    <p>
+        <span class="rfc2119-assertion" id="td-model-protocol-security-restriction">
+    Since a <a>Thing Model</a> SHOULD NOT contain complete <a>Protocol Binding</a> information such as endpoint address and/or
+    specific security mechanism.
```

Comment:
Again, I DO want to specify the security schemes if they apply to the entire class of instances the model describes.

Reply:
This depends on the use cases (digital twin use case vs. developer use case). Then let's go back to MAY NOT to cover different use cases. @mlagally is it also ok for you?

Reply:
done